### ISSUE TITLE
Import old forum content

### ISSUE THREAD

**Issue by Deleetdk** [openpsych/django#78]
Some notes:

- Rename the submission threads to delete the journal tag, e.g. [ODP]. We will handle this in a proper way and so the title no longer needs these.

**Comment by hmcclungiii** [comment]
Is this only applicable to the new forums, or should the titles be changed in the old MyBB forums as well?

**Comment by Deleetdk** [comment]
Only the new.

**Comment by hmcclungiii** [comment]
Assumptions are a bad thing, usually, when left untested during development.  There is an implied assumption here.  

The assumption here is that there are no other systems or scripts out there that are linked to the old forums, and dependent on the current titles in the database.  Cron jobs, SERPs.  Activity comparison scripts.  Anything.

It is highly unlikely that this assumption is false, except for one small caveat.  Organic SERP's, those you've seen on Google Webmaster Console, Analytics and maybe AdWords.  To take a loss on any significant marketing position, if any, you've gained thus far in this niche should be frowned upon.

@Deleetdk Please validate this assumption with your insight.

**Comment by hmcclungiii** [comment]
Revisiting this now that I've finished the import.

Remove all these prefixes from all the thread titles, correct?
![screenshot 68 _li](https://user-images.githubusercontent.com/6778244/51495649-17e42c80-1d8b-11e9-9711-d7dba0e87ed4.jpg)

Have you considered my previous comment?

**Comment by Deleetdk** [comment]
The prefixes denote which journal the submission is for. However, this information should be handled as part of properly encoded metadata, not just something users add manually in the titles.

Making threads in the submissions forum has to be handled differently than other forum. There should be a different form to fill out that has fields for the information every study must have. Title, abstract, link to materials, author list, journal submitted to. When we this created and are happy with the result, we can manually parse the old submissions to fill in this information into the right fields. There's something like 65 submissions, so it shouldn't take so long to do this.

So, yes, this info should eventually be removed from the titles but only after the above has been done.

**Comment by hmcclungiii** [comment]
The only journals that should be available are those listed on the Journals page of the site, correct?

**Comment by Deleetdk** [comment]
I've made a new issue for this submissions forum extra functionality: https://github.com/openpsych/django/issues/90

**Comment by hmcclungiii** [comment]
One question about when the user selects the journal for their submission:

Can the user select more than one journal for submission?

In other words, is this a many-to-one relationship, or a many-to-many?

**Comment by hmcclungiii** [comment]
@hmcclungiii 

>Can the user select more than one journal for submission?

> >No, a paper can only be in one journal, as currently.

Ok, so we're going for a many-to-one relationship here.  

There are two instances where the journal prefix is not enclosed with brackets [...].  So the import must match for the prefixes without enclosing brackets.

Also, I noticed there are prefixes in the meta forum that aren't related to the journals.  These prefixes are
* [statistics]
* [in queue]
* [bug]

So I'm going to assume that this sort of functionality will be needed as well.  If so, I'll create a new issue, if not, I'll just leave them as they are.

**Comment by hmcclungiii** [comment]
commit 8508104844ba2fcfec1dabe326ae824e2e86cc83 fixes & closes #78

**Comment by Deleetdk** [comment]
This issue is clearly not complete. For instance, attached files are missing.

https://openpsych.net/forum/reputation.php?uid=1
https://beta.openpsych.net/thread/4865

**Comment by hmcclungiii** [comment]
I'm closing this issue because I created a new issue #98, which covers the attached files and may require discussion indirectly related to importing MyBB, deserving its own space.
